.github/workflows/build.yml

```diff
@@ -5,7 +5,7 @@ on:
   push:
     branches:
       - main
-      - 'release-[0-9].[0-9]+'
+      - 'release-0.11.1'
```

Comment:
This is good to trigger the workflows here. Generally we don't want to create `release-x.x.x` branches for fixpatch releases so we keep this branch rule regex in the `release-0.11.1` branch.